test/templates/button-group/index.njk

```diff
@@ -52,8 +52,65 @@
         }}
     {{-  button(
           { label: 'Third' },
+            modifier={ block: [size] })
+          }}
+  {%- endcall %}
+
+
+    {% endset %}
+    {{ format(example) }}
+
+
+    {% endfor %}
+
+<h1> disabled </h1>
+<h2>default </h2>
+
+
+    {% for size in ["","compact"] %}
+      {%- set modifier = { block: [size] } %}
+      {%- if size == "default" %}
+        {%- set modifier = { block: [] } %}
+      {%- endif %}
+      
+<h2> {{size}} </h2>
+
+{% set example %}
+
+  {% call button_group() -%}
+    {{-  button(
+      { icon: 'survey' },
+      disabled = true,
+        modifier={ block: [size] },
+        aria={ label: 'Survey' })
+      }}
+    {{-  button(
+        { icon: 'pie-chart' },
+          modifier={ block: [size] },
+          aria={ label: 'Chart' })
+        }}
+    {{-  button(
+          { icon: 'pool' },
             modifier={ block: [size] },
-            aria={ })
+            aria={ label: 'Pool' })
+          }}
+  {%- endcall %}
+
+  {% call button_group() -%}
+    {{-  button(
+      { label: 'First' },
+      disabled = true,
+        modifier={ block: [size] },
+        aria={ })
+      }}
+    {{-  button(
+        { label: 'Second' },
+          modifier={ block: [size] },
+          aria={ })
+        }}
+    {{-  button(
+          { label: 'Third' },
+            modifier={ block: [size] })
```

Comment:
Example wise, there should be examples that show different buttons as the disabled version (not always the first one).